src/game.zig

```diff
@@ -126,6 +124,20 @@ pub fn run() void {
     var gfx_state = gfx.init(std.heap.page_allocator, main_window) catch unreachable;
     defer gfx.deinit(&gfx_state, std.heap.page_allocator);
 
+    var prefab_manager = pm.PrefabManager.init(&flecs_world, std.heap.page_allocator);
+    defer prefab_manager.deinit();
+
+    var arena_state = std.heap.ArenaAllocator.init(std.heap.page_allocator);
+    defer arena_state.deinit();
+    const arena = arena_state.allocator();
+    zmesh.init(arena);
+    defer zmesh.deinit();
+
+    // TODO(gmodarelli): Add a function to destroy the prefab's GPU resources
+    spider_prefab = prefab_manager.loadPrefabFromGLTF("content/prefabs/creatures/spider/spider.gltf", &flecs_world, &gfx_state, std.heap.page_allocator) catch unreachable;
```

Comment:
Just a thhought for the future - I think the prefabmanager should register callbacks for handling various components of the prefab